pentaho-karaf-features/src/main/resources/pentaho-karaf-features-standard.xml

```diff
@@ -248,4 +248,28 @@
     <bundle dependency="true" start-level="30">mvn:org.ow2.asm/asm-all/5.0.2</bundle>
     <bundle start-level="30">mvn:org.eclipse.jetty.aggregate/jetty-all-server/8.1.15.v20140411</bundle>
   </feature>
+    <!-- Overriding activemq to exclude xtreaam 1.4.7 version  -->
+  <feature name="activemq" description="ActiveMQ broker libraries" version="5.12.0" resolver="(obr)" start-level="50">
+        <feature>http</feature>
+        <feature version="5.10.0">activemq-client</feature>
+        <bundle>mvn:org.apache.activemq/activemq-karaf/5.10.0</bundle>
+        <bundle dependency="true">mvn:org.apache.xbean/xbean-spring/3.16</bundle>
+        <bundle dependency="true">mvn:commons-collections/commons-collections/3.2.2</bundle>
+        <bundle dependency='true'>mvn:commons-lang/commons-lang/2.6</bundle>
+        <bundle dependency="true">mvn:commons-codec/commons-codec/1.4</bundle>
+        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.velocity/1.7_6</bundle>
+        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jasypt/1.9.1_1</bundle>
+        <bundle dependency="true">mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.stax-api-1.0/1.9.0</bundle>
+        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.xpp3/1.1.4c_5</bundle>
+        <bundle dependency="true">mvn:joda-time/joda-time/1.6.2</bundle>
+        <!--<bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.xstream/1.4.7_1</bundle>-->
```

Comment:
@CodeOnCoffee says that he is certain that this isn't being used by this feature and is OK with it being commented out.